std_msgs/Header header
float32 bus_voltage
float32 bus_current
float32 temperature
